network.c

```diff
@@ -735,8 +735,10 @@ static int network_open(const struct iio_device *dev,
 		goto out_mutex_unlock;
 
 	ret = create_socket(pdata->addrinfo, DEFAULT_TIMEOUT_MS);
-	if (ret < 0)
+	if (ret < 0) {
+		ERROR("Create socket: %d\n", ret);
```

Comment:
Just a minor thing: I saw that "Create socket" and "Open unlocked" have lower case but the rest of messages are all uper case.